razorpay/client.py

```diff
@@ -80,6 +82,12 @@ def _get_version(self):
             pass
         return version
 
+    def _set_app_details(self, app_details):
+        self.app_details.append(app_details)
+
+    def _get_app_details(self):
```

Comment:
app_details are not getting used anywhere, refer to this
https://github.com/razorpay/razorpay-php/blob/master/src/Request.php#L157